src/main/java/com/facebook/openwifirrm/optimizers/tpc/LocationBasedOptimalTPC.java

```diff
@@ -153,27 +153,29 @@ public static List<Integer> runLocationBasedOptimalTPC(
 	/**
 	 * Calculate new tx powers for the given band.
 	 *
-	 * @param band       band
+	 * @param channel channel
+	 * @param serialNumbers The serial numbers of the APs with the channel
 	 * @param txPowerMap this map from serial number to band to new tx power
 	 *                   (dBm) must be passed in empty, and it is filled in by
 	 *                   this method with the new tx powers.
 	 */
-	private void buildTxPowerMapForBand(
-		String band,
+	private void buildTxPowerMapForChannel(
+		int channel,
+		List<String> serialNumbers,
```

Comment:
The list of serialNumbers are computed by `TPC.getApsPerChannel`, and I believe there would be no repetitions if our input radio state is good. Set is better in terms repetition as you said, but since we only use this collections for looping, a list is more efficient in both time and space in this use case.

Reply:
fair point! Yeah it was a minor comment more than anything